### PR TITLE
closes #149, closes #150  and closes #152

### LINKED ISSUE #149: Add Open Classifieds to CMS
http://open-classifieds.com/

https://github.com/open-classifieds/openclassifieds2

### LINKED ISSUE #150: Add Open eShop to E-commerce
https://github.com/open-classifieds/open-eshop/

### LINKED ISSUE #152: Add Jekyll to Blogging Platforms and CMS
http://jekyllrb.com/

### PR DESCRIPTION
**Pull request by neo22s**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,6 +201,7 @@ CMS are a practical way to setup a website with many features. CMS often come wi
   * [MODX](http://modx.com/) - MODX is an advanced content management and publishing platform. The current version is called 'Revolution'. ([Source Code](https://github.com/modxcms/revolution)) `GPLv2` `PHP`
   * [Movable Type](https://movabletype.org/) - a powerful solution for the creation and management of web content.
   * [october](http://octobercms.com/) - Free, open-source, self-hosted CMS platform based on the Laravel PHP Framework. ([Source code](https://github.com/octobercms/october)) `MIT`
+  * [Open Classifieds](http://open-classifieds.com/) - Free open-source, self-hosted CMS for classifieds sites. ([Source code](https://github.com/open-classifieds/openclassifieds2))  `GPL` `PHP`
```
Comment (nodiscc) [inline README.md]: Shouldn't this be in the E-commerce subsection?

File: README.md [hunk 2]
```diff
@@ -86,6 +86,7 @@ See also [Static Site Generators](#static-site-generators) and  [Content Managem
   * [Dotclear](http://dotclear.org/) - Take control over your blog! ([Source code](https://hg.dotclear.org/dotclear)) `GPLv2` `PHP`
   * [Ghost](http://ghost.org/) - Just a blogging platform ([Source code](https://github.com/TryGhost/Ghost)) `MIT` `Nodejs`
   * [htmly](http://www.htmly.com/) - Databaseless Blogging Platform (Flat-File Blog) ([Source code]((https://github.com/danpros/htmly)), [Demo](http://www.htmly.com/demo/)) `GPLv2` `PHP`
+  * [Jekyll] (http://jekyllrb.com/) - Simple, blog-aware, static sites. ([Source code](https://github.com/jekyll/jekyll) `MIT` `Ruby`
```
Comment (nodiscc) [inline README.md]: We don't list static site generators, please see https://github.com/Kickball/awesome-selfhosted/issues/72, https://github.com/Kickball/awesome-selfhosted/issues/66, https://github.com/Kickball/awesome-selfhosted/issues/41. Can you remove this entry?

**Comment by nodiscc** [comment]
Thank you @neo22s , can you edit your patch to address the 2 issues mentioned above? I may be wrong about Open Classifieds, if you think it really fits in CMS, please let me know.

File: README.md (hunk 1 above)
Comment (neo22s) [inline README.md]: mmm you are probably right! I will thanks ;)

**Comment by neo22s** [comment]
done! https://github.com/Kickball/awesome-selfhosted/pull/154

**Pull request closed without merging**